spec/services/order_builder_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe PurchasesHandler::OrderBuilder, type: :service do
+
+  describe "#call" do
```

Comment:
```suggestion
  describe ".call" do
```

Por convenção utilizamos `.` para indicar que o método é estático